plexmuxy/config.py

```
@@ -0,0 +1,379 @@
+    if not extension.startswith("."):
+        extension = f".{extension}"
+    return extension
+
+
+def bool_value(value: Any, field_name: str) -> bool:
+    if isinstance(value, bool):
+        return value
+    raise ConfigError(f"{field_name} must be a boolean")
+
+
+def thread_count(value: Any, field_name: str) -> int | str:
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)